### PR TITLE
Distributed tracing blog post

### PR DESCRIPTION
**Pull request by deltaepsilon**
image

### CODE REVIEW

File: blog-content/tracing-distributed-systems-in-nextjs.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+tags: Tutorial
+metaTitle: "Tracing Distributed Systems in Next.js"
+---
+
+
+[![Watch on YouTube](/images/blog/tracing-distributed-systems-in-nextjs/distributed-tracing-livestream-16x9-play.png)](https://youtube.com/live/z9g-eOPwndk)
+
+## Distributed Tracing
+
+A naive approach to tracing creates a new trace for every service that is used to resolve a request.
```
Comment (SpennyNDaJets) [inline blog-content/tracing-distributed-systems-in-nextjs.md:22]: May just be github font, but had a tough time working out this sentence in my head. Once I read it a few times it makes sense, but wasn't sure if you noticed it. Possibly the use of `tracing` and `trace` so close together? Not blocking, but wanted to raise

**Review by SpennyNDaJets — APPROVED** [review]

File: blog-content/tracing-distributed-systems-in-nextjs.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+All three services, the web client, the Next.js API function, and the Golang backend, feed their data to the same Highlight project.
+
+The result can be seen on the Traces tab of the Highlight dashboard.
+
+This example shows four separate spans, all of which are nested under a single trace. And of course, these traces are accessible from Highlight's Session Replay tab, because the session was generated in the web client and propagated throughout the entire distributed "call stack".
+
+- `highlight-run-with-headers` is generated by the `AppRouterHighlight` wrapper function in Next.js.
+- `fetch GET https://localhost:3010/test` is generated by the Next.js `Fetch` request.
+- `highlight.chi` is created by Highlight's chi middleware.
+- `go-custom-span` is a custom span that we used in our example code for demonstration purposes. It wraps a call to `time.Sleep(1 * time.Second)`.
```
Comment (Vadman97) [inline blog-content/tracing-distributed-systems-in-nextjs.md:198]:
```suggestion
- `go-custom-span` is a custom span that we used in our example code for demonstration purposes. It wraps a call to `time.Sleep(1 * time.Second)` to mimic slow code execution.
```

File: blog-content/tracing-distributed-systems-in-nextjs.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+```jsx
+ <button
+    onClick={async () => {
+        await fetch("/propagation-test");       
+    }}
+>
+    Trigger propagation
+</button>
+```
```
Comment (Vadman97) [inline blog-content/tracing-distributed-systems-in-nextjs.md:103]: Screenshot 2024-04-18 at 16 50 44

this is doing something weird with the styling

**Review by Vadman97 — APPROVED** [review]
nice! write up is great

**Comment by deltaepsilon** [comment]
Here's a current screenshot:
image

**Pull request merged**